### PR TITLE
Release to slack [MAILPOET-1885]

### PR DESCRIPTION
**Pull request by JanJakes**

### CODE REVIEW

File: tasks/release/ChangelogController.php [hunk 1]
```diff
@@ -38,8 +38,9 @@ function get($version_name = null) {
     if (!$changelog) {
       $changelog = self::FALLBACK_RECORD;
     }
-    $notes = $this->renderList($issues, Jira::RELEASENOTE_FIELD_ID);
-    return [$heading, $changelog, $notes];
+    $notes = $this->renderList($issues, JiraController::RELEASENOTE_FIELD_ID);
+    $release_id = $version['id'];
+    return [$heading, $changelog, $notes, $release_id];
```
Comment (costasovo) [inline tasks/release/ChangelogController.php]: Do we need to add $release_id into this output? I can't see it used anywhere.

Comment (JanJakes) [reply tasks/release/ChangelogController.php]: Good catch, I originally used output of this method until I realized I need to fetch version info directly from JiraController anyway. So I guess it's forgotten here. Will fix.

**Comment by JanJakes** [comment]
@costasovo I think we can add it in the last ticket when dealing with naming and command grouping. If I add it now it will be shared _before_ release is uploaded on GitHub.

**Comment by JanJakes** [comment]
Fixed.

**Pull request merged**